packages/shared/src/components/fields/TextField.tsx

```diff
@@ -18,7 +18,7 @@ import BaseFieldContainer, {
 import { ButtonProps } from '../buttons/Button';
 import useInputFieldFunctions from '../../hooks/useInputFieldFunctions';
 
-export interface TextFieldProps extends BaseFieldProps<HTMLInputElement> {
```

Comment:
Why did we end up removing the type parameter?

Reply:
It showed as ambiguous from the global search

Reply:
Yeah, makes sense. TS would have complained right after this if it should not have happened.